lib/qtrajectory.h

```diff
@@ -55,6 +55,29 @@ class QuantumTrajectorySimulator {
      * If true, normalize the state vector before performing measurements.
      */
     bool normalize_before_mea_gates = true;
+    /**
+     * If false, do not apply deferred operators after the main loop.
+     * This can be used to speed up simulations of circuits with weak noise
+     * and without measurements by reusing the results for the "primary" noise
+     * realization, that is the noise realization in which the  "primary"
+     * (highest probability) Kraus operators are sampled in each channel.
+     * The primary Kraus operators should be first in their respective channels.
+     */
+    bool apply_last_deferred_ops = true;
```

Comment:
Does setting this to false reduce performance in the general case (i.e. non-weak noise)?

Reply:
No, it doesn't reduce performance.

Reply:
This will produce invalid results in the general case because collecting and reusing the required results might not be implemented (as, say, in pybind_interface/pybind_main.cpp).

Reply:
Got it, this makes sense now. I think it's really important to be clear that this is not a "flip the boolean for better performance" situation - let's separate out the line that calls this out:
```
/**
 * <existing comment, minus the line below>
 *
 * It is the client's responsibility to collect the primary trajectory results
 * and to reuse them.
 *\
```

Reply:
Done.